packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker-task-base-types.ts

```diff
@@ -193,17 +231,20 @@ export interface ResourceConfig {
 }
 
 /**
+ * Specifies the VPC that you want your Amazon SageMaker training job to connect to.
  *
  * @experimental
  */
 export interface VpcConfig {
     /**
-     * VPC id
+     * VPC ID
```

Comment:
It's not the VPC id. It's the vpc itself.

Reply:
will remove the id